### PR TITLE
Move labels to i18n resource

### PR DESCRIPTION
**Pull request by reda-idtaleb**
The labels are moved to the i18n resources in the both sides (backend and frontend).

Here's some screenshots (on JMeter and on the web) that showing the labels in French :
i18n2
i18n1

### CODE REVIEW

File: back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/listener/gui/ULPObservabilityGui.java [hunk 1]
```diff
@@ -130,32 +188,32 @@ private void init() {
      */
     private JPanel createSamplerConfigPanel() {
     	JPanel ulpObservabilityConfigPanel = new JPanel(new MigLayout("wrap 2", "[][fill,grow]0px"));
-    	ulpObservabilityConfigPanel.setBorder(BorderFactory.createTitledBorder("Config"));
+    	ulpObservabilityConfigPanel.setBorder(BorderFactory.createTitledBorder(MessageUtils.getMessage("config")));
```
Comment (pmouawad) [inline back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/listener/gui/ULPObservabilityGui.java:191]: I prefer this approach to the one creating a constant for label key

File: back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/util/MessageUtils.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	
+	/**
+     * Get the corresponding key value in the bundle resource 
+     * whose base name is defined by the BUNDLE_BASE_NAME constant.
+     * The key value is retrieved based on the JMeter locale.
+     * @param key the key of a resource bundle
+     * @return the value of the key.
+     */
+    public static String getMessage(String key) {
+    	ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE_BASE_NAME, JMeterUtils.getLocale());
```
Comment (pmouawad) [inline back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/util/MessageUtils.java]: Can you add Thread.currentThread().getContextClassLoader() as last parameter of getBundle(...) ?

File: back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels.properties
Comment (pmouawad) [inline back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels.properties:1]: Can you run dos2unix on the file as there are ^M at end of each line

File: back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels_en.properties
Comment (pmouawad) [inline back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels_en.properties:1]: Same thing here

File: back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels.properties
Comment (pmouawad) [inline back/src/main/resources/com/ubikloadpack/jmeter/ulp/observability/i18n/guiLabels.properties:1]: Can you also rename guiLabels_XX.properties to messages_XX.properties

File: ulp-observability-front/src/main/front/package.json [hunk 3]
```diff
@@ -35,6 +38,7 @@
     "@angular-devkit/build-angular": "^13.3.6",
     "@angular/cli": "~13.1.2",
     "@angular/compiler-cli": "~13.1.0",
+    "@angular/localize": "^13.1.3",
```
Comment (pmouawad) [inline ulp-observability-front/src/main/front/package.json]: Is this needed ?

File: ulp-observability-front/src/main/front/src/app/app.module.ts [hunk 4]
```diff
@@ -53,9 +55,23 @@ export class MaterialModule {};
     NgChartsModule,
     BrowserAnimationsModule,
     MaterialModule,
-    FlexLayoutModule
+    FlexLayoutModule,
+    TranslateModule.forRoot({
+      defaultLanguage: 'fr',
```
Comment (pmouawad) [inline ulp-observability-front/src/main/front/src/app/app.module.ts]: defaultLanguage should be en

File: back/licenses/bsd 2-clause "simplified" license - license.html
Comment (pmouawad) [inline back/licenses/bsd 2-clause "simplified" license - license.html:1]: Can you revert this change ?

File: ulp-observability-front/src/main/front/src/polyfills.ts [hunk 5]
```diff
@@ -1,3 +1,7 @@
+/***************************************************************************************************
+ * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
+ */
+import '@angular/localize/init';
```
Comment (pmouawad) [inline ulp-observability-front/src/main/front/src/polyfills.ts]: Is this needed ?

File: ulp-observability-front/src/main/front/src/assets/i18n/i18n.babel
Comment (pmouawad) [inline ulp-observability-front/src/main/front/src/assets/i18n/i18n.babel:1]: Is this needed ?

File: ulp-observability-front/src/main/front/src/app/app.module.ts (hunk 4 above)
Comment (pmouawad) [reply ulp-observability-front/src/main/front/src/app/app.module.ts]: Maybe you need to add something so that the language is taken from server side ? 
There is a config url that may need to be enriched get the language from  JMeterUtils.getLocale() on the client side. And you would then adapt code to take it into account on client side

**Comment by pmouawad** [comment]
Hello @reda-idtaleb , is the PR ready ? Can I merge it ? Thanks

**Comment by reda-idtaleb** [comment]
Yes I tested it and it's ok.
